invenio_app_rdm/records_ui/searchapp.py

```diff
@@ -0,0 +1,230 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018-2021 CERN.
+#
+# Invenio-App-RDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Configuration helper for React-SearchKit."""
+
+from copy import deepcopy
+from functools import partial
+
+from flask import current_app
+from invenio_rdm_records.searchconfig import OptionsSelector, \
+    SortOptionsSelector
+
+
+class SortConfig(SortOptionsSelector):
+    """Sort options for the search configuration."""
+
+    def map_option(self, key, option):
+        """Generate a RSK search option."""
+        return {
+            "sortBy": key,
+            "text": option['title']
+        }
+
+
+class FacetsConfig(OptionsSelector):
+    """Facets options for the search configuration."""
+
+    def map_option(self, key, option):
+        """Generate an RSK aggregation option."""
+        title = option.get('title', option['facet']._label)
+
+        ui = deepcopy(option['ui'])
+        ui.update({
+            'aggName': key,
+            'title': title,
+        })
+
+        # Nested facets
+        if 'childAgg' in ui:
+            ui['childAgg'].set_default('aggName', 'inner')
+            ui['childAgg'].set_default('title', title)
+
+        return ui
+
+
+class SearchAppConfig:
```

Comment:
Taken from Invenio-Search-UI

https://github.com/inveniosoftware/invenio-search-ui/blob/master/invenio_search_ui/views.py#L59

Reply:
Much seems to be the same than `SearchAppInvenioRestConfigHelper` but in a slightly different manner. i.e. every method has one more line or one more option, many tiny changes.

Are we implying we are moving away from `invenio-search-ui`? 

If not, is it worth to create a common interface? It is just a bit weird to have two classes that do *almost exactly the same* in different modules. Maybe we can find a middle ground or migrate `invenio-shearch-ui` to the new one?

This can be shelved (i.e. open an issue) since we are in a rush for releasing.

Reply:
> Are we implying we are moving away from invenio-search-ui?

Yes. There's other problems like it's not applying proper permissions checks in search-ui and other in flexiiblities that means it's not worthwhile continue using it. Similar to that we removed usage of records-ui.